content/millennium/r4/financial/charge-item.md

```diff
@@ -45,6 +45,7 @@ The following fields are returned if valued:
 * [Performing Location]
 * [Procedure]
 * [Revenue Code]
+* [Replacing]
```

Comment:
nit: `Replacing` should be before revenue code

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/626/commits/9db66e25714bc87e3463b174c9adb8e5506567d0#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR48)